easybuild/easyconfigs/h/h5py/h5py-2.10.0-foss-2020a-Python-3.8.2.eb

```diff
@@ -30,4 +30,8 @@ download_dep_fail = True
 # to really use mpi enabled hdf5 we now seem to need a configure step, which is the reason we can't use pip
 prebuildopts = 'python setup.py configure --mpi --hdf5=$EBROOTHDF5 && '
 
+# skip regular 'import h5py', and run it via mpirun (since 'import h5py' results in an MPI_Init call)
+options = {'modulename': False}
+sanity_check_commands = ["mpirun -np 1 python -c 'import h5py'"]
+
```

Comment:
```suggestion
# skip regular 'import h5py', and run it via mpirun (since 'import h5py' results in an MPI_Init call)
options = {'modulename': False}
sanity_check_commands = ["mpirun -np 1 python -c 'import h5py'"]

```

Reply:
I prefer using the same sanity check command in both `foss` and `intel` easyconfigs.

This may actually be strictly required, for example in a Slurm environment.

I'll wait until @ocaisa's PR https://github.com/easybuilders/easybuild-framework/pull/3264 is merged, and then use `"%(mpi_cmd_prefix)s python -c 'import h5py'"` here, so I can configure EasyBuild to use `srun --ntasks %(nr_ranks)s %(cmd)s` as MPI command template...

Reply:
I'll also test this PR with the new Intel MPI 2019.7 as soon as possible